plutus-ledger/src/Ledger/Params.hs

```diff
@@ -99,6 +99,7 @@ instance Default ProtocolParameters where
     , protocolParamMaxValueSize = Just 5000
     , protocolParamCollateralPercent = Just 150
     , protocolParamMaxCollateralInputs = Just 3
+    , protocolParamUTxOCostPerByte = Nothing
```

Comment:
```suggestion
    , protocolParamUTxOCostPerByte = Just (Lovelace 4311)
```
(protocolParamUTxOCostPerWord / 8)

Reply:
Hm, then it should be `4310.25`? `34482/8`

Reply:
We don't have fractional Lovelace. I found 4310 mentioned in a cardano-wallet PR.

Reply:
But only update it if you get test failures, otherwise I should do it in my Babbage PR.